### DISCUSSION TITLE
Control plane with CAX11 type

### DISCUSSION THREAD

**Discussion by Stringls** [syself/cluster-api-provider-hetzner#791] [General]
Hi! Is it possible to initialize a control plane with `CAX11` type of the server?
When I was generating template I've set `HCLOUD_CONTROL_PLANE_MACHINE_TYPE` to `CAX11`, but in the KCP I have `AMD64` packages, while in MD I have `$ARCH`.

So as I understand right now we can only have workers with `CAX11` while CP is with `AMD64` arch?

**Comment by batistein** [comment]
You can use cax11 for control-planes. The templates packaged in each release are not suited for production-use. We didn't integrated e2e test for control-planes with arm this is why it's missing the $ARCH variable.

**Reply by Stringls** [reply]
> You can use cax11 for control-planes. The templates packaged in each release are not suited for production-use. We didn't integrated e2e test for control-planes with arm this is why it's missing the $ARCH variable.

1. What do you mean by "not suited for production-use"?
2. So I can just update the template for KCP and I'll be able to use CAX11 for CP?
